config/deploy.rb

```diff
@@ -42,7 +40,8 @@
     to :launch do
       queue "mkdir -p #{deploy_to}/#{current_path}/tmp/"
       queue "touch #{deploy_to}/#{current_path}/tmp/restart.txt"
-      invoke :'unicorn:restart'
+      queue "#{rake} unicorn:restart"
+      queue "#{rake} sidekiq:restart"
```

Comment:
This doesn't work though :grin: 

We need to require those tasks from `config/deploy.rb` and use `invoke` here